### DISCUSSION TITLE
Transfering reports from Datapane cloud to Github

### DISCUSSION THREAD

**Discussion by unknownned** [datapane/datapane#389] [Help]
### Summary

As noted in the latest blog update the Datapane cloud will close down. It is mentioned that it is possible to upload the reports to your own Github environment. I cannot seem to find information on how to do this in the docs section. Please advice how to move the reports from Datapane cloud to Github.

**Comment by XinyiWang-Jessica** [comment]
upload the html file as index.html and enable the GitHub Page. It works for me.

**Comment by mands** [comment]
Hi,

It's not possible to export reports from Datapane Cloud. Instead the best way is to rerun the local Python code that created the report, then use either `dp.save_report` to generate a single file or `dp.build_report` to create a directory bundle, and copy the file/directory into your GH pages repo.
